psi4/src/psi4/libmints/wavefunction.h

```diff
@@ -574,10 +574,13 @@ class PSI_API Wavefunction : public std::enable_shared_from_this<Wavefunction> {
 
     /// Returns the SO basis Lagrangian
     SharedMatrix lagrangian() const;
-    /// Returns the Lagrangian in SO basis for the wavefunction
-    SharedMatrix X() const;
```

Comment:
I can't do the code suggestion op on removed lines, I guess, but I think this needs to come back and have the `PSI_DEPRECATED(` too.

Reply:
I see. I have it back. For consistency, I also add the `Lagrangian` to the Py-side marked as deprecated.